packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -4115,6 +4115,31 @@ class _TestWidgetInspectorService extends TestWidgetInspectorService {
     testWidgetsWithLeakTracking('ext.flutter.inspector.show', (WidgetTester tester) async {
       final Iterable<Map<Object, Object?>> extensionChangedEvents = service.getServiceExtensionStateChangedEvents('ext.flutter.inspector.show');
       Map<Object, Object?> extensionChangedEvent;
+      int debugShowChangeCounter = 0;
+
+      final GlobalKey key = GlobalKey();
+      await tester.pumpWidget(
+        WidgetsApp(
+          key: key,
+          builder: (BuildContext context, Widget? child) {
+            return const Placeholder();
+          },
+          color: const Color(0xFF123456),
+        ),
+      );
+
+      final ValueListenableBuilder<bool> valueListenableBuilderWidget = find
+          .byType(ValueListenableBuilder<bool>)
+          .evaluate()
+          .single
+          .widget as ValueListenableBuilder<bool>;
```

Comment:
Could this be simplified to:

```suggestion
      final ValueListenableBuilder<bool> valueListenableBuilderWidget = tester.widget(
        find.byType(ValueListenableBuilder<bool>),
      );
```

Reply:
Maybe it has to be tester.widget<ValueListenableBuilder<bool>>(...) to make the type system happy.